### DISCUSSION TITLE
Continuous Test: why can I only set either basic-auth or tenant_id?

### DISCUSSION THREAD

**Discussion by oujonny** [grafana/mimir#6066] [Help and support]
We recently enabled basic auth on the gateway, then the continuous test does not work anymore because we forgot about the auth part there. I was able to configure it with basic_auth, but why can't I then specify a tenant? I would like to use basic auth and the test metrics for a specific tenant (not "anonymous")

see code here: https://github.com/grafana/mimir/blob/c13a9aefd598973c0f66785d876021369bcfaa3d/pkg/continuoustest/client.go#L102C2-L102C2

**Comment by dimitarvdimitrov** [comment]
i opened https://github.com/grafana/mimir/issues/6091 to change the behaviour either in continuous test or in the nginx config.
